src/runner.rs

```diff
@@ -25,6 +24,43 @@ pub fn runner(requirements: &[Requirements]) -> ExitCode {
     conclusion.exit_code()
 }
 
+fn find_tests(args: &Arguments, requirements: &[Requirements]) -> Vec<Trial> {
+    let tests: Vec<_> = if let Some(exact_filter) = exact_filter(args) {
+        let exact_tests: Vec<_> = requirements
+            .iter()
+            .flat_map(|req| req.exact(exact_filter))
+            .collect();
+
+        match exact_tests.len() {
+            0 if is_nextest() => {
```

Comment:
Could this just have an outer `if is_nextest()` condition?

Reply:
We could, and that would be fine.
I wrote it like this because it does the cheap part (length comparison) first and the (relatively) expensive part (env var string comparison) second. It's never going to matter performance-wise to just always check the environment variable. I'll do it that way.